python/tvm/tir/schedule/schedule.py

```diff
@@ -442,6 +442,64 @@ def after_split(a: ty.handle, b: ty.handle) -> None:
         # that there is at most one None in `factors`
         return _ffi_api.ScheduleSplit(self, loop, factors)  # type: ignore # pylint: disable=no-member
 
+    def reorder(self, *loops: List[LoopRV]) -> None:
```

Comment:
To be consistent with the C++ API

```suggestion
    def reorder(self, *ordered_loops: List[LoopRV]) -> None:
```